File: taller2/ParOImpar.java
import java.util.Scanner;
// Determinar si un número ingresado por teclado es par o impar


public class ParOImpar {
     // Variable de recoleccion de datos
    private static Scanner readScanner = new Scanner(System.in);

    // Metodo de identificación de datos positivo o negativo
public static String identificaNumeroParOImpar(Double num){
    if(num%2 == 0){
        return "El número ingresado es par.";
    }else if (num%2 != 0) {
        return "El número ingresado es impar.";
    }
    
    return "Ingreso un dato erroneo";
}

// Metodo de ejecución del proyecto
    public static void main(String[] args) {
        System.out.println("Ingrese un numero");
        Double num = readScanner.nextDouble();
        String resParImpar = identificaNumeroParOImpar(num);
        System.out.println(resParImpar);
    }
}

File: taller2_py/par_o_impar.py
# Solicita al usuario que ingrese un número y lo convierte a tipo flotante
num = float(input("Ingrese un número: "))

# Verifica si el número es par o impar
if num % 2 == 0:  # Si el residuo de la división entre 2 es 0, es un número par
    print("El número ingresado es par.")
elif num < 0:  # Si el el numero es negativo es un dato erroneo
    print("Ingreso un dato erróneo")
else: 
    print("El número ingresado es impar.") 
